### PR TITLE
Conversation API WeChat and Telegram removal and fixes in Verify

### PR DESCRIPTION
**Pull request by SpyrosChr**
1) Remove Telegram from API reference
 2) Remove WeChat from API reference
 3) Create a new postman collection without WeChat and Telegram
 3) Fix typos and schemas in Verify

### CODE REVIEW

**Review by hdd-24 — APPROVED** [review]

File: verify/v1/verify.yaml [hunk 1]
```diff
@@ -487,7 +487,7 @@ paths:
           content:
             application/json:
               schema:
-                anyOf:
+                oneOf:
                   - $ref: '#/components/schemas/VerifyResponse'
                   - $ref: '#/components/schemas/ValidNumberFormatFalseResponse'
```
Comment (priyankaupadhyay090) [inline verify/v1/verify.yaml:492]: with oneOf, can we see the rest of responses in actual conversation?

File: verify/v1/verify.yaml [hunk 2]
```diff
@@ -504,6 +504,27 @@ paths:
                   - $ref: '#/components/schemas/MccMncMismatchResponse'
                   - $ref: '#/components/schemas/DisposableNumberFalseResponse'
                   - $ref: '#/components/schemas/DisposableNumberTrueResponse'
+              examples:
+                success:
+                  summary: Example of a successful response
+                  value:
+                    requestId: "4457068c-194d-4f72-9550-13b81ec054a9",
```
Comment (priyankaupadhyay090) [inline verify/v1/verify.yaml:511]: this line is giving error while trying to convert the content of yaml into online swagger UI (same in my local). Is it running in your local?

File: verify/v1/verify.yaml (hunk 1 above)
Comment (SpyrosChr) [reply verify/v1/verify.yaml:492]: no but this should be one of...
oneof means that only one of the responses will be sent back
anyof means that multiple of the responses can be sent back

https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

So I think oneof is the correct option

File: verify/v1/verify.yaml (hunk 2 above)
Comment (SpyrosChr) [reply verify/v1/verify.yaml:511]: you are right... my mistake.
It is the coma at the end of the values

**Pull request merged**